app/Resources/views/plant/show.html.twig

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div class="row">
+        <div class="col-md-12">
+            <div class="article">
+                <h1>Details of the plant</h1><br>
+                <div class="table-responsive">
+                    <table class="table table-hover">
+                        <tr>
+                            <th>Name</th>
+                            <td>{{ plant.name }}</td>
+                        </tr>
+                        <tr>
+                            <th>Description</th>
+                            <td>{{ plant.description }}</td>
+                        </tr>
+                        <tr>
+                            <th>Place</th>
+                            <td>{{ plant.place }}</td>
+                        </tr>
+                        <tr>
+                            <th>When</th>
+                                {% if plant.frequency==1 %}
+                            <td>{ plant.frequencyPerDay }} times a day</td>
```

Comment:
this does not show properly because on the left side one brace is missing.
is: `{ plant.frequencyPerDay }}`
should be: `{{ plant.frequencyPerDay }}`